templates/forms/UserFormsCheckboxSetField.ss

```diff
@@ -0,0 +1,11 @@
+<% if $Options.Count %>
+    <% loop $Options %>
+        <div class="$Class">
+            <input id="$ID" class="checkbox" name="$Name" type="checkbox" value="$Value.ATT"<% if $isChecked %>
```

Comment:
This was deliberate yes so as to be consistent with the CheckboxSetField.ss within the framework.